doc/modules/tree.rst

```diff
@@ -345,6 +345,10 @@ Tips on practical use
     :ref:`ICA <ICA>`, or :ref:`feature_selection`) beforehand to
     give your tree a better chance of finding features that are discriminative.
 
+  * :ref:`sphx_glr_auto_examples_tree_plot_unveil_tree_structure.py` will help
+    in gaining more insights about the nodes which can have an influence over the
```

Comment:
"how a tree makes predictions" might be clearer than "the nodes which can have an influence ..."